src/index.ts

```diff
@@ -31,10 +32,19 @@ class DXScannerCommand extends Command {
     const scanPath = args.path || process.cwd();
     cli.action.start(`Scanning URI: ${scanPath}`);
 
-    const container = createRootContainer({ uri: scanPath });
+    const container = createRootContainer({ uri: scanPath, auth: authorization });
     const scanner = container.get(Scanner);
 
-    await scanner.scan();
+    try {
+      await scanner.scan();
+    } catch (error) {
+      authorization = await cli.prompt('Insert your GitHub personal access token.\nhttps://github.com/settings/tokens\n');
+
+      const container = createRootContainer({ uri: scanPath, auth: authorization });
```

Comment:
I am not sure if it is necessary to create new container... Could we just set credentials to `ArgumentsProvider` already existing in the container before? It could be better because of file caching etc.